src/printer.js

```diff
@@ -1475,9 +1475,43 @@ function genericPrintNoParens(path, options, print, args) {
         parts.push(print(childPath));
 
         if (i < expressions.length) {
+          // For a template literal of the following form:
+          //   `someQuery {
+          //     ${call({
+          //       a,
+          //       b,
+          //     })}
+          //   }`
+          // the expression is on its own line (there is a \n in the previous
+          // quasi literal), therefore we want to indent the JavaScript
+          // expression inside at the beginning of ${ instead of the beginning
+          // of the `.
+          let size = 0;
+          const value = childPath.getValue().value.raw;
+          const index = value.lastIndexOf('\n');
+          const tabWidth = options.tabWidth;
+          if (index !== -1) {
+            for (let i = index + 1; i < value.length; ++i) {
+              if (value[i] === '\t') {
+                // Tabs behave in a way that they are aligned to the nearest
+                // multiple of tabWidth:
+                // 0 -> 4, 1 -> 4, 2 -> 4, 3 -> 4
+                // 4 -> 8, 5 -> 8, 6 -> 8, 7 -> 8 ...
+                size = Math.floor(((size + tabWidth) / tabWidth) * tabWidth);
```

Comment:
No it's not, I floor it in between. So if `size === 3`, it's going to be `4` and not `7`.

Your computation does the same thing and is smaller, i'll use it instead :) Thanks!